packages/dioxus-blitz/src/documents/dioxus_document.rs

```diff
@@ -338,14 +570,48 @@ impl WriteMutations for MutationWriter<'_> {
     }
 
     fn set_node_text(&mut self, value: &str, id: ElementId) {
-        let node_id = self.state.element_to_node_id(id);
-        let node = self.doc.get_node_mut(node_id).unwrap();
-        if let NodeData::Text(ref mut text) = node.raw_dom_data {
-            text.content = value.to_string();
+        let parent = {
+            let node_id = self.state.element_to_node_id(id);
+            let node = self.doc.get_node_mut(node_id).unwrap();
+
+            if let NodeData::Text(ref mut text) = node.raw_dom_data {
+                text.content = value.to_string();
+            }
+
+            node.parent
+        };
+
+        println!("set_node_text id:{} text:{}", id.0, value);
+
+        if let Some(parent) = parent {
+            // if the text is the child of a style element, we want to put the style into the stylesheet cache
+            let parent = self.doc.get_node(parent).unwrap();
+            if let NodeData::Element(ref element) = parent.raw_dom_data {
+                if element.name.local.as_ref() == "style" {
+                    self.doc.add_stylesheet(value);
```

Comment:
This probably also needs to remove the existing stylesheet (if any). Otherwise mutations to stylesheets are going to end up creating duplicates